src/components/playerdash/player-card.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import utils from '../../utils/utilities.js';
+import CardLine from './player-card-table-line';
```

Comment:
The only issue I have with `CardLine` as a name is if you want to use the component again later in a different context it may not make sense. I think it's all good for now. If it gets reused at a later date we can refactor and name it something universal and move it from the `playerdash` dir. It could probably be extended to be `row` fairly easily and be passed an array of fields so it could be any length. No need to worry about doing that now though.. :smile: